packages/manager/modules/anthos/src/dashboard/storage/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,13 @@
+{
+  "anthos_dashboard_storage_description": "Vous pouvez décomposer l’espace de stockage disponible sur votre Dedicated NetApp en plusieurs Volumes SVM. Vous pourrez associer ces volumes SVM à un déploiement Anthos spécifique, et les utiliser en tant que CSI prédéfini (Container Storage Interface).",
```

Comment:
```suggestion
  "anthos_dashboard_storage_description": "Vous pouvez décomposer l’espace de stockage disponible sur votre cluster NetApp dédié en plusieurs Volumes SVM. Vous pourrez associer ces volumes SVM à un déploiement Anthos spécifique, et les utiliser en tant que CSI prédéfini (Container Storage Interface).",
```

Reply:
@ganeshkumar1989, can you please change the sentence.
It was requested from reporter.

Thanks.

Reply:
Thanks @mohammed-zahaf . This is done :)